go.mod

```diff
@@ -0,0 +1,13 @@
+module kubevirt.io/project-infra/limiter
+
+go 1.11
+
+require (
+	cloud.google.com/go v0.39.0
+	github.com/joshdk/go-junit v0.0.0-20190428045703-ad7e11aa49ff
+	github.com/onsi/ginkgo v1.7.0 // indirect
+	github.com/onsi/gomega v1.4.3 // indirect
+	github.com/sirupsen/logrus v1.4.2
+	google.golang.org/api v0.5.0
```

Comment:
I believe I have seen this issue before. The issue was related to the version mismatch.
Since we have switched to golang modules, it might happen.

I would propose to add 
```
google.golang.org/grpc v1.19.0
```
here. And see whether it helps. It should help go mod to figure out the minimal version required.

Reply:
Also please make sure all bazel caches are clean. :)

Reply:
Thanks for your help, @petrkotas! 

I suppose I just should add the line in `go.mod`, right? 

I understood from reading the go modules manual that you would normally do a `go get google.golang.org/grpc@v1.19.0`, but that produced a lot of other entries? However, will try out and tell how it goes. Thanks again!

Reply:
not sure, maybe it has to go into replaces section, since this is not required directly by us, but should be overridden in our dependencies IIUC?

Reply:
AFAIK when you do not include the direct version, `go mod` will scan through the code dependencies, it will find there is grpc required, will select minimum usable version and will include it.

If you include the minimum version yourself, you are forcing `go mod` to use the one you require.

Reply:
Tried it, didn't help.

I got feedback from the bazel mailing list pointing me towards [rules_go - Avoiding conflicts](https://github.com/bazelbuild/rules_go/blob/master/proto/core.rst#id5) (proto related paragraph) and [this related GitHub issue](https://github.com/bazelbuild/rules_go/issues/1548). Am still trying to understand and integrate this, could still need some help.

Reply:
Just for reference, the solution has [been described here](https://github.com/bazelbuild/bazel-gazelle/issues/432#issuecomment-457789836).